include/uxr/agent/datareader/DataReader.hpp

```diff
@@ -34,6 +34,7 @@ class DataReader : public XRCEObject
     static std::unique_ptr<DataReader> create(
         const dds::xrce::ObjectId& object_id,
         const std::shared_ptr<Subscriber>& subscriber,
+        const std::shared_ptr<ProxyClient>& proxy_client,
```

Comment:
May we could use the `subscriber_id` instead of the `subscriber` itself, it should be more efficient.

Reply:
Done